code/game/objects/effects/anomalies.dm

```diff
@@ -484,17 +477,11 @@
 		// Add target
 		affected += target
 
-		// Insert a random organ
-		var/obj/item/organ/picked_organ = pick(ANOMALY_DELIMBER_ZONE_ORGANS)
-		var/obj/item/organ/new_organ = new picked_organ
-		new_organ.Insert(target, TRUE, FALSE)
-
 		// Replace a random limb
 		var/picked_zone = pick(ANOMALY_DELIMBER_ZONES)
 		var/obj/item/bodypart/picked_user_part = target.get_bodypart(picked_zone)
 		if(!picked_user_part)
 			target.update_body(TRUE)
```

Comment:
Updating no longe rneeds to happen since we never insert anything?

Reply:
We replace limbs. This proc updates damage overlays for the bodyparts and stuff.

Reply:
This proc is executed when no limbs are replaced